app/test_main.py

```diff
@@ -1,3 +1,25 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (100, 100, [21, 17]),
+        (-1, -1, [0, 0]),
```

Comment:
Testing with negative values is good for edge cases, but the expected result should be carefully considered. If negative values are not valid input, the function should ideally raise an exception or handle it differently rather than returning [0, 0].